apps/org/utils.py

```diff
@@ -17,8 +20,25 @@ def set_unique_slug(instance, based_on_field='name'):
     slug = slugify(getattr(instance, based_on_field))
     # If this slug is already being used, create a more unique slug
     if instance.__class__.objects.filter(slug=slug).exists():
-        slug = '{}-{}'.format(
-            slug,
-            ''.join(random.choices(string.ascii_letters + string.digits, k=20))
-        )
+        slug = '{}-{}'.format(slug, ''.join(
+            random.choices(string.ascii_letters + string.digits, k=20)))
     instance.slug = slug
+
+
+def make_qr_code(text, format='PNG', encoding='base64'):
```

Comment:
A namespace for this could be something like `libs.qr_code.make_qr_code` allowing other modules to cleanly depend on it.

Reply:
Okay, I created a "libs" package inside the smh_app repo and put this in libs/qrcode.py. Tests also moved there.